src/common/options.cc

```diff
@@ -821,7 +821,7 @@ std::vector<Option> get_global_options() {
     .set_description(""),
 
     Option("ms_dpdk_coremask", Option::TYPE_STR, Option::LEVEL_ADVANCED)
-    .set_default("1")
+    .set_default("0xF")         //begin with 0x for the string
```

Comment:
since you are at here, could you add a `.add_see_also("ms_async_op_threads")` to emphasis the relation between these two options?

Reply:
updated.